.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Needs to be ignored in .gitattributes

Reply:
I thought so too, but apparently it's already covered:
https://github.com/phpstan/phpstan/blob/c77503fe47fb306f215e5b28a161242cf4cb0512/.gitattributes#L1
right?